editor/extensions/ext-placemark.js

```diff
@@ -0,0 +1,513 @@
+/**
+ * ext-placemark.js
+ *
+ *
+ * @copyright 2010 CloudCanvas, Inc. All rights reserved
+ *
```

Comment:
This is fine, but just want to confirm, you have the rights to this and are allowing the code to be released under the MIT license, correct?